src/test/java/io/reactivex/rxjava3/schedulers/IoScheduledReleaseTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2016-present, RxJava Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivex.rxjava3.schedulers;
+
+import io.reactivex.rxjava3.core.Completable;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.core.RxJavaTest;
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class IoScheduledReleaseTest extends RxJavaTest {
+
+    /* This test will be stuck in a deadlock if rx3.io-scheduled-release is not set */
+    @Test
+    public void scheduledRelease() {
+        System.setProperty("rx3.io-scheduled-release", Boolean.TRUE.toString());
```

Comment:
This will not work because the classes have been already initialized. For a workaround, change

`private static final boolean USE_SCHEDULED_RELEASE;`

to

`static boolean USE_SCHEDULED_RELEASE;`

move this `IoScheduledReleaseTest` to `package io.reactivex.rxjava3.internal.schedulers;`,

then:

```java
boolean save = IoScheduler.USE_SCHEDULED_RELEASE;
IoScheduler.USE_SCHEDULED_RELEASE = true;
try {
  // ...
} finally {
    IoScheduler.USE_SCHEDULED_RELEASE = save;
}
```

Reply:
I don't know how it was running on my machine, but this makes sense, 👍